x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/service/CreateServiceAccountTokenResponseTests.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.security.action.service;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.common.settings.SecureString;
+import org.elasticsearch.test.AbstractWireSerializingTestCase;
+
+import java.io.IOException;
+
+public class CreateServiceAccountTokenResponseTests extends AbstractWireSerializingTestCase<CreateServiceAccountTokenResponse> {
+
+    @Override
+    protected Writeable.Reader<CreateServiceAccountTokenResponse> instanceReader() {
+        return CreateServiceAccountTokenResponse::new;
+    }
+
+    @Override
+    protected CreateServiceAccountTokenResponse createTestInstance() {
+        return CreateServiceAccountTokenResponse.created(
+            randomAlphaOfLengthBetween(3, 8), new SecureString(randomAlphaOfLength(20).toCharArray()));
+    }
+
+    @Override
+    protected CreateServiceAccountTokenResponse mutateInstance(CreateServiceAccountTokenResponse instance) throws IOException {
+        return CreateServiceAccountTokenResponse.created(randomAlphaOfLengthBetween(3, 8),
+            new SecureString(randomAlphaOfLength(20).toCharArray()));
```

Comment:
There doesn't seem to be any guarantee (other than luck) that this instance will be different than the one passed in.
```suggestion
        return randomBoolean()
        ? CreateServiceAccountTokenResponse.created(randomAlphaOfLength(10), instance.getValue())
        : CreateServiceAccountTokenResponse.created(instance.getName(), new SecureString(randomAlphaOfLength(22).toCharArray()))
        ;
```

Reply:
I thought I have consistently used `randomValueOtherThan`. Apparently I didn't I will update to useit.